  Fixed bug in detecting old-style iterables (PY-4890)  